---
title: "ChromoSeq: Introduction for Trainees"
format: html
editor: visual
theme: flatly
toc: false
---

```{=html}
<style>title {text-align: left}</style>
<style>body {text-align: justify}</style>
```
## Overview

[ChromoSeq](https://pathologyservices.wustl.edu/services-tests/chromoseq/) is a whole-genome sequencing assay used to comprehensively identify multiple classes of clinically relevant genetic variation---including copy number alterations, structural variants, and gene mutations---in patients with myeloid malignancies ([PMID: 33704937](https://pubmed.ncbi.nlm.nih.gov/33704937/)).

## Workflow

ChromoSeq sequencing runs are started on Monday and Friday, and output data are copied to the Department of Pathology server the following Thursday or Monday. There is a standing appointment for faculty sign out at 14:00 on Tuesdays and Fridays, which may be moved or canceled depending on faculty and data availability.

Once data are available, for each case:

-   Evaluate the overall quality of the data based on the genome-wide coverage plot (`*genomewide.pdf`), quality control metrics (`ChromoSeq QC`), and haplotect output
-   Review the patient history in detail (diagnostic and treatment history, prior molecular and cytogenetic testing results, corresponding morphological case, corresponding cytogenetic and molecular testing results, etc.)
-   Use the `ChromReporter` tool to:
    -   Enter information about the case as structured data

    -   Update the variants included in the clinical report based on variant quality, orthogonal test results, and patient history

    -   Provide a concise summary of key findings and an overall assessment of the case

    -   Save the updated data as a JSON and overwrite the existing `*chromoseq.json` file
-   Review the case with sign out faculty
-   Revise the case and notify faculty when it is ready to be signed out

## Data

The complete set of files for each case (including alignment files) are stored on the Research Infrastructure Services (RIS) cluster ([ris.wustl.edu](https:\ris.wustl.edu)). The subset of files needed to interpret most cases are copied to the Department of Pathology server (pathfiles.wustl.edu).

To access data on a Windows machine, use the File Explorer application:

-   Right click on `This PC` and select `Map network drive...`
-   Type `\\pathfiles.wustl.edu/AMP/Genomics/ChromoSeq` into the `Folder` box and click `Finish`

To access data on a Mac, use the Finder application:

-   Select `Go` → `Connect to Server` from the menu bar (or press `⌘K`)
-   Type `smb://pathfiles.wustl.edu/AMP/Genomics/ChromoSeq` into the `Server Address` box and click `Connect`

You will need both read and write access to the `batchdir/` directory to template cases. If you do not have both, ask one of the fellows to help arrange access.

Data from individual sequencing runs are stored in the `batchdir/` directory and named by run date. Data from individual cases are stored in subfolders named by patient MRN, CoPath accession, and specimen type. There should be five files for each case:

-   `*genomewide.pdf`: genome-wide coverage plot applying all appropriate corrections; used to visually assess copy number alterations
-   `*genomeWideCorrection.pdf`: genome-wide coverage plots applying different subsets of corrections (or no correction); used to visually assess copy number alterations
-   `*chromoseq.txt`: human-readable summary of variants identified and quality control metrics; used as an alternative source of data also available via the `ChromReporter` tool
-   `*chromoseq.json`: machine-readable summary of variants identified, quality control metrics, and case information; used by the `ChromReporter` tool to generate the clinical report
-   `*haplotect.txt`: haplotect output; used to assess sample contamination and/or donor engraftment if clinically relevant

## Reporting Tool

The `ChromReporter` reporting tool is a JavaScript application developed and maintained by Ajay Khanna at the McDonnell Genome Institute. The application can be launched in a web browser (Chrome, Safari, etc.) by navigating to the `ChromoSeq/ChromReporter/` directory and clicking on the `index.html` file.

The `ChromReporter` tool reads data from the `*chromoseq.json` files associated with each case. Clicking on `Choose File` and then selecting a JSON will bring up four tabs displaying the information stored for an individual case:

-   `Case Information`: contains structured fields to input case information and relevant clinical data as well as a comment field to provide a narrative interpretation for each case
-   `ChromoSeq Data`: displays variants called by the ChromoSeq analysis pipeline; allows users to promote or remove variants from the report based on review of the underlying sequencing data and correlation with orthogonal studies
-   `ChromoSeq QC`: displays case information and quality control metrics as formatted tables
-   `Raw Data`: displays the contents of the associated `*chromoseq.txt` file as plain text

After selecting variants to report, entering structured data, and writing the case interpretation, clicking on `View PDF` generates a formatted PDF report for review. From this page, clicking `View JSON` generates a JSON file containing all of the case information, including user-entered data. To save work/update a case in progress, save the JSON with the same name and in the same location as the original file.

Note: the `Submit to CoPath` checkbox is **only used by faculty** when signing out cases. This box should never be checked by trainees templating cases.

## Quality Control Review

Quality control metrics are summarized under the `ChromSeq QC` tab. Cases with any of the following metrics outside of the reference range may require additional sequencing, preparation of new sequencing libraries, or submission of alternative specimens:

| Metric                                | Reference Range |
|:--------------------------------------|----------------:|
| Average Coverage (X)                  |           \>40X |
| Total Reads (10<sup>9</sup>)          |             \>1 |
| Total Base Pairs (10<sup>9</sup>)     |           \>150 |
| Mapped Reads (%)                      |            \>95 |
| Duplicate Reads (%)                   |            \<40 |
| Unique Reads (%)                      |            \>60 |
| Properly Paired Reads (%)             |            \>95 |
| Mean Insert Size (bp)                 |           \>300 |
| Haplotect (informative sites \>0) (%) |              ≤1 |

## Structured Data Fields

-   `Ordering Physician`: attending physician first and last name as they appear on the associated Epic order
-   `Last Name`: patient last name as it appears in Epic
-   `First Name`: patient first name as it appears in Epic
-   `Middle Name`: patient middle name as it appears in Epic
-   `Corresponding Morphological Case`: CoPath accession for the corresponding bone marrow biopsy
-   `Blast Count (%)`: blast count (%) recorded in the differential count from the corresponding bone marrow biopsy
-   `WBC (K/cumm)`: white blood cell count (K/cumm) recorded in the complete blood count (CBC) from the corresponding bone marrow biopsy
-   `Hgb (g/dL)`: hemoglobin (g/dL) recorded in the complete blood count (CBC) from the corresponding bone marrow biopsy
-   `Plt (K/cumm)`: platelet count (K/cumm) recorded in the complete blood count (CBC) from the corresponding bone marrow biopsy
-   `Sign-out Pathologist`: Eric Duncavage or Molly Schroeder
-   `Stated Diagnosis`: AML, CML, MDS, MPN, MDS/MPN, or other
-   `Diagnosis timepoint`: new diagnosis, recurrence, or other
    -   In general, patients recently diagnosed at an outside institution who are being referred for evaluation without prior treatment should be counted as new diagnoses
-   `Transplant`: allogeneic, autologous, or NA
-   `Prior history of MDS`: yes, no, or unknown
-   `Prior history of cancer`: yes, no, or unknown
-   `NCCN AML Risk Category`: favorable, intermediate, or poor/adverse (new diagnosis of non-APL AML only)
-   `IPSS-R MDS Cytogenetic Risk Category`: very good, good, intermediate, poor, or very poor (new diagnosis of MDS only)
-   `Comment`: result interpretation

## Updating Reported Variants

Use the checkboxes next to individual variants under the `ChromoSeq Data` tab to include or remove them from the clinical report.

In general, the set of variants included by default are the appropriate set to report. Possible exceptions include:

**Copy Number Alterations**

-   A clinically relevant copy number alteration is included under filtered variants and is clearly visible on the `*genomewide.pdf` coverage plot
-   A clinically relevant copy number alteration is included under filtered variants, is visible on the `*genomewide.pdf` coverage plot, and has been identified by orthogonal studies in the same patient

**Structural Variants**

-   A clinically relevant structural variant is included under filtered variants, the variant call appears to be valid (based on coverage, a contig that [blats](https://genome.ucsc.edu/cgi-bin/hgBlat) cleanly to the identified regions, etc.), and the variant was reported by cytogenetics or there is high suspicion for the variant given the clinical presentation

**Gene Mutations**

-   A gene mutation is included under filtered variants, the variant call appears to be valid (based on coverage, review in [IGV](https://software.broadinstitute.org/software/igv/), etc.), and the variant was reported by a previous or the corresponding MyeloSeq study

## Interpretation

Case interpretations are provided in the `Comment` field and should be tailored to the specific variants identified, patient history, and associated clinical data. The goal is to clearly and concisely describe key results, comment on consistencies/discrepancies with orthogonal molecular and cytogenetic testing, and provide a comprehensive summary of how the results impact the patient's diagnosis and overall risk stratification, if appropriate.

The following statements (in order) may be useful for templating comments in some cases but will require modification for accuracy, clarity, and readability. Note that the interpretive comments included in cytogenetic reports provide excellent examples for how to appropriately describe the cytogenetic alterations enumerated by [International System for Human Cytogenomic Nomenclature](https://iscn.karger.com) (ISCN) karyotypes.

-   *(No copy number alterations \| A \[deletion \| duplication\] of the \[long \| short\] arm of chromosome \[chr\] \| No copy number alterations) (was \| were) identified, consistent with (a complex karyotype and) the corresponding cytogenetic studies (G22-####).*
-   *(In addition, a \[translocation \| inversion \| interchromosomal rearrangement \| intrachromosomal rearrangement\] involving the \[(short \| long) arm of chromosome (chr) and the (short \| long) arm of chromosome (chr)\] was identified.)*
-   *(\[A mutation in (gene) \| Mutations in (genes)\] \[was \| were\] also detected, consistent with the corresponding MyeloSeq study (W22-####).)*
-   *(We note that \[a mutation in (gene) \| mutations in (genes)\] detected by MyeloSeq at \[a VAF \| VAFs\] below 10% \[was \| were\] not identified by ChromoSeq, although this is below the limit of detection of this assay.)*
-   *(Given the patient history and associated clinical data, this case is best classified as \[classification\] by International Consensus Classification (ICC) criteria.)*
-   *(The overall constellation of mutations is consistent with \[risk group\] molecular risk per National Comprehensive Cancer Network (NCCN) guidelines.)*

### References

1.  Arber DA *et al*. International Consensus Classification of Myeloid Neoplasms and Acute Leukemias: integrating morphologic, clinical, and genomic data. *Blood*. 2022 Sep 15;140(11):1200-1228. [PMID: 35767897](https://pubmed.ncbi.nlm.nih.gov/35767897/)

2.  Duncavage EJ *et al*. Genome Sequencing as an Alternative to Cytogenetic Analysis in Myeloid Cancers. *N Engl J Med*. 2021 Mar 11;384(10):924-935. [PMID: 33704937](https://pubmed.ncbi.nlm.nih.gov/33704937/)

3.  Greenberg PL *et al*. Myelodysplastic Syndromes, Version 2.2017, NCCN Clinical Practice Guidelines in Oncology. *J Natl Compr Canc Netw*. 2017 Jan;15(1):60-87. [PMID: 28040720](https://pubmed.ncbi.nlm.nih.gov/28040720/)

4.  Greenberg PL *et al*. NCCN Guidelines Insights: Myelodysplastic Syndromes, Version 3.2022. *J Natl Compr Canc Netw*. 2022 Feb;20(2):106-117. [PMID: 35130502](https://pubmed.ncbi.nlm.nih.gov/35130502/)

5.  Papaemmanuil E *et al*. Genomic Classification and Prognosis in Acute Myeloid Leukemia. *N Engl J Med*. 2016 Jun 9;374(23):2209-2221. [PMID: 27276561](https://pubmed.ncbi.nlm.nih.gov/27276561/)

6.  Pollyea DA *et al*. NCCN Guidelines Insights: Acute Myeloid Leukemia, Version 2.2021. *J Natl Compr Canc Netw*. 2021 Jan 6;19(1):16-27. [PMID: 33406488](https://pubmed.ncbi.nlm.nih.gov/33406488/)

7.  Tallman MS *et al*. Acute Myeloid Leukemia, Version 3.2019, NCCN Clinical Practice Guidelines in Oncology. *J Natl Compr Canc Netw*. 2019 Jun 1;17(6):721-749. [PMID: 31200351](https://pubmed.ncbi.nlm.nih.gov/31200351/)
