---
title: "MyeloSeq: Introduction for Trainees"
format: html
editor: visual
theme: flatly
---

```{=html}
<style>title {text-align: left}</style>
<style>body {text-align: justify}</style>
```
### Overview

[MyeloSeq](https://pathologyservices.wustl.edu/items/myeloseq/) is a high-depth tumor-only targeted sequencing panel designed to identify recurrent gene mutations in patients with myeloid neoplasms. MyeloSeq may be performed as part of a patient's initial diagnosis (where it supports disease classification, treatment selection, and risk stratification) as well as for monitoring. MyeloSeq's limit of detection (LOD) for identifying new variants is 2.0% variant allele fraction (VAF). In addition, MyeloSeq's LOD for detecting previously identified mutations is 0.1%, which is sufficient for measurable residual disease (MRD) assessment.

### Workflow

MyeloSeq is currently run once per week. Sequencing libraries are loaded on a NovaSeq X Plus on Monday, and data are typically available for analysis by Wednesday morning. Trainees review the sequencing data and template clinical reports for sign out by 1400 on Friday afternoon. Briefly, trainees are expected to:

-   Summarize clinically relevant information in a concise history
-   Review quality control data and notify the fellow on service of any cases that don't meet target thresholds
-   Evaluate and classify individual variants
-   Provide a brief clinical interpretation for the overall set of mutations identified
-   Upload case data to CoPath and format the clinical report
-   Notify the fellow and/or attending on service of any questions or concerns related to the technical details of the case and/or the clinical interpretation of the data

### Data

You will need read access to the Department of Pathology server as well as an active CoPath account to prepare cases. Please reach out to the fellow on service to arrange access if needed. Accessing these resources from off campus will require you to be on the medical school virtual private network (VPN).

To connect to the department server on a Windows computer, use the File Explorer application:

-   Right click on `This PC` and select `Map network drive...`
-   Type `\\pathfiles.wustl.edu/AMP/Genomics/Myeloseq_Results` into the `Folder` box and click `Finish`

To access data on a Mac, use the Finder application:

-   Select `Go` → `Connect to Server` from the menu bar (or press `⌘K`)
-   Type `smb://pathfiles.wustl.edu/AMP/Genomics/Myeloseq_Results` into the Server Address box and click Connect

Cases are organized on the department server by batch. Each batch directory contains a separate folder for each case as well as a TSV file summarizing the quality control metrics for all cases (`QC_metrics.tsv`). Each case folder contains the following files:

-   `*annotated.vcf.gz`: variant call format file of all variants detected by the analysis pipeline
-   `*annotated_filtered.vcf.gz`: variant call format file of filtered variants
-   `*cram`: alignment file
-   `*cram.crai`: alignment index
-   `*report.json`: case report text file
-   `*report.txt`: case report json file

### QC Review

The first step in reviewing a case is to ensure that the sequencing data meet the validated quality thresholds. These data are included at the top of each `*report.txt` file along with reference ranges:

``` bash
MAPPING/ALIGNING SUMMARY: Total bases (Gbp)     15.0    >12
MAPPING/ALIGNING SUMMARY: Total input reads (M) 134.1   >100
MAPPING/ALIGNING SUMMARY: Q30 bases R1 (%)      95.9    >90
MAPPING/ALIGNING SUMMARY: Q30 bases R2 (%)      94.4    >90
MAPPING/ALIGNING SUMMARY: Mismatched bases R1 (%)       0.22    <0.9
MAPPING/ALIGNING SUMMARY: Mismatched bases R2 (%)       0.24    <0.9
MAPPING/ALIGNING SUMMARY: Mapped reads (%)      99.6    >98
MAPPING/ALIGNING SUMMARY: Insert length: mean   147.0   >120
COVERAGE SUMMARY: Aligned reads in target region (%)    74.7    >70
COVERAGE SUMMARY: Target bases >250x (%)        99.4    >96
COVERAGE SUMMARY: Target bases >1250x (%)       95.0    >85
COVERAGE SUMMARY: Average alignment coverage over target region 4370.7  >2500
UMI SUMMARY: On target mean family depth        7.8     >5 <15
UMI SUMMARY: Families discarded (%)     20.4    <30
AMPLICON SUMMARY: Amplicons with low coverage (%)       6.1     <50
```

The most important metrics to focus on are `COVERAGE SUMMARY: Average alignment coverage over target region`, `COVERAGE SUMMARY: Target bases >250x (%)`, and `MAPPING/ALIGNING SUMMARY: Total bases (Gbp)`. For a new diagnosis, mean coverage needs to be \>1,000x and \>96% of the target space needs to acheive \>250x coverage. For a negative MRD case, mean coverage needs to be \>2,500x, and positions with previously identified mutations need to be covered at \>2,500x.

Cases that don't meet the mean coverage and/or coverage breadth targets and also don't have adequate total bases may benefit from additional sequencing of the existing library. Cases that don't meet the mean coverage and/or coverage breadth targets and do have adequate total sequencing may indicate a failed sequencing library, requiring a new library to be made and/or an alternative specimen to be analyzed.

It is also important to review the haplotect summary for every case:

``` bash
*** Haplotect Contamination Estimate ***

totalSites      informativeSites        meanCoverage    contaminatingHaplotypeCount     totalReads      contaminationEstimate
75      2       1881.24 4.0     7582    0.0011
```

Haplotect is an algorithm that analyzes known polymorphic sites in the genome to quantitatively estimate sample contamination (i.e., evidence of the presence of DNA from more than one individual in the sequencing data). A `contaminationEstimate` \>0.02 (2%) is concerning for contamination. However, in patients who have received an allogeneic hematopoietic cell transplant, elevated Haplotect scores most likely represent mixed chimerism, which should be noted in the case interpretation.

### Reporting Tool

We use a javascript reporting tool to help template cases: `Genomics/Myeloseq_Results/MyeloReporter`. Clicking `index.html` opens the application in your default web browser (e.g., Chrome, Safari, etc.). To load a case, click `Choose File` and select the corresponding `*report.json` file. Alternatively, in most browsers, you can drag the JSON file on top of the `Choose File` button to upload it. Uploading the file pulls in detailed information for all variants identified by the analysis pipeline and generates several fields for data entry: `Variant Interpretation`, `Clinical History`, and `Specimen Site`.

### Patient History

Please provide a concise history summarizing clinically relevant patient information. Histories should not be copy-pasted from clinical notes and should not contain any undefined abbreviations. Key data to review are:

-   Recent **clinical notes** to place the specimen in the appropriate clinical context and treatment timepoint (notes from the Bone Marrow Transplant service are often the most informative for this purpose)
-   The corresponding **surgical pathology case**
-   The corresponding **cytogenetic studies**

Here, corresponding means from the same bone marrow biopsy, if applicable. You will also need to review prior MyeloSeq and cytogenetic studies to determine which variants are being followed in patients who are being monitored. In addition, it can be helpful to check for measurable residual disease testing performed by flow cytometry and/or the results of STR identity testing in patients who have received allogeneic hematopoietic cell transplants, but these data generally do not need to be included in the history unless there are specific points of confusion they address.

In general, the following structure is appropriate for most cases:

*(##)-year-old (man\|woman) with a history of (myeloid neoplasm (diagnosed YYYY-MM-DD) with \[a normal karyotype\|a complex karyotype\|{short list of cytogenetic abnormalities}\] \[and mutations in {gene list}\]), who (is currently being treated and\|is status post hematopoietic cell transplant (YYYY-MM-DD) and) (presents for evaluation\|presents for treatment\|presents for follow up). Corresponding bone marrow biopsy (S##-####) shows (top-line diagnosis). Corresponding cytogenetic studies (G##-####) show ({list of FISH findings}\|karyotype\|no evidence of clonal aberrations). Corresponding ChromoSeq study (W##-###) (did not identify any copy number alterations, structural variants, or gene mutations\|showed {list of variants}).*

### Variant Review

Within the reporting tool, variants are organized into the following sections:

-   `Reportable Mutations`: new variants detected at a VAF \>2% or prior variants detected at a VAF \>0.1%
-   `Prior Mutations Not Detected in This Case`: prior variants not detected at \>0.1% VAF
-   `Low-Level Somatic Mutations`: variants that passed the quality thresholds defined by the analysis pipeline but that were present below the validated VAF thresholds
-   `Filtered Variants (Report with Caution)`: variants that failed quality thresholds defined by the analysis pipeline; these should almost never be reported
-   `Tier 4 Polymorphisms`: variants that passed the quality thresholds defined by the analysis pipeline and that have been reported at \>0.1% of individuals in population databases

In general, the default set of variants selected for reporting is correct. However, linking to historical cases is based on patient identifier lists that are not always complete, meaning sometimes the set of previously detected variants is missing of incomplete. Please confirm that previously reported variants (based on your review of prior MyeloSeq cases) appear in either the `Reportable Mutations` or `Prior Mutations Not Detected in This Case` tables. In some cases, missing variants can be recovered from the `Low-Level Somatic Mutations` table. Other times, this requires additional investigation. Please let the fellow on service know if you encounter a case that does not appear to be correctly linked to previous studies.

If you have reason to believe that a particular variant is an artifact, you can remove it from the report by unchecking the `Report` or `Report + previously detected` box associated with each record.

Variants that appear in `Tier 4 Polymorphisms` should almost never be included in the report. The exception to this guideline is if there is a specific clinical concern for inherited cancer predisposition in a particular patient and a known pathogenic/likely pathogenic variant appears in this table. This most often occurs in relation to *DDX41* variants. Patients with *DDX41*-mutated myeloid neoplams typically have one germline and one somatic *DDX41* variant. Some pathogenic germline *DDX41* variants have been seen in the population at \>0.1% minor allele frequency and, therefore, may appear in the `Tier 4 Polymorphisms` table.

### Variant Classification

The current version of the reporting tool classifies individual variants as pathogenic or of uncertain significance based on whether they have been reported by one of several high-quality sequencing studies involving large cohorts of patients with myeloid neoplasms. We have modified this classification for our clinical reports, but this action is not yet supported within the reporting tool. Therefore, variant classifications have to be manually updated after exporting the data from the reporting tool. Please ask the fellows for a current version of the variant classification standard operating procedure for details regarding the variant classification rules. Briefly, variants should be classified as one of:

-   `Potentially therapeutic`: variant associated with an FDA-approved targeted therapy (*FLT3*-ITD, *IDH1* R132, *IDH2* R140, and *IDH2* R172)
-   `Pathogenic`: not potentially therapeutic and previously reported by high quality sequencing studies of patients with myeloid neoplasms or a recurrent class of mutations in patients with myeloid neoplasms as defined by National Comprehensive Cancer Network (NCCN) guidelines
-   `Possibly donor-derived`: not potentially therapeutic or pathogenic, found at a VAF of 35%--65% in a patient after hematopoietic cell transplantation
-   `Possibly germline`: not potentially therapeutic or pathogenic, found at a VAF of 35%--65% in a patient with no history of hematopoietic cell transplantation
-   `Uncertain`: all other variants

### Case Interpretation

The goal of the case interpretation is to summarize the clinical significance of the overall set of mutations observed in the case with an emphasis on their collective implications for diagnosis, treatment, and risk stratification.

For new diagnoses of myelodysplastic syndrome and acute myeloid leukemia, this should include the disease classification according to the [International Consensus Classification (ICC)](https://pubmed.ncbi.nlm.nih.gov/35767897/) critera as well as [the 5th edition of the World Health Organization Classification of Haematolymphoid Tumours (WHO-HAEM5)](https://pubmed.ncbi.nlm.nih.gov/35732831/) when possible. In addition, an overall risk stratification consistent with the [National Comprehensive Cancer Network (NCCN)](https://www.nccn.org/guidelines/category_1) guidelines should be provided for new diagnoses.

Examples:

-   *Mutations in PTPN11, RUNX1, and U2AF1 are identified. This case is best classified as acute myeloid leukemia with myelodysplasia-related gene mutations by International Consensus Classification (ICC) criteria or acute myeloid leukaemia, myelodysplasia-related by the 5th edition of the World Health Organization Classification of Haematolymphoid Tumours (WHO-HAEM5). The overall constellation of mutations is consistent with high risk acute myeloid leukemia by National Comprehensive Cancer Network (NCCN) guidelines.*

-   *A TP53 c.713G\>A (p.C238Y) mutation is identified. This case is best classified as myelodysplastic syndrome with mutated TP53 by International Consensus Classification (ICC) criteria, or myelodysplastic neoplasm with biallelic TP53 inactivation by the 5th edition of the World Health Organization Classification of Hematolymphoid Tumors (WHO-HAEM5). The overall constellation of mutations is consistent with high risk myelodysplastic syndrome.*

For patients who have an existing diagnosis and are being monitored, the interpretation usually consists entirely of an assessment of MRD status, i.e., MRD positive or MRD negative.

Examples:

-   *Previously identified mutations in FLT3 and IDH1 are again seen, consistent with persistent molecular disease (measurable residual disease \[MRD\] positive). There is evidence of significant chimerism.*

-   *Previously identified mutations in ASXL1, DNMT3A, and TP53 are not seen, consistent with mutation clearance and measurable residual disease (MRD) negative status at a sensitivity of at least 0.1%. Overall quality control metrics are within normal limits. There is no evidence of significant chimerism.*

-   *Previously identified leukemia-associated mutations in CEBPA, FLT3, NPM1, and WT1 are not seen, consistent with mutation clearance and measurable residual disease (MRD) negative status at a sensitivity of at least 0.1%. Overall quality control metrics are within normal limits. We note that a TET2 c.3409+1G\>T mutation is again seen at a reduced variant allele fraction compared to this patient's prior study. Persistent TET2 mutations after treatment are not associated with an increased risk of relapse in patients with acute myeloid leukemia (PMID: 29601269).*

For patients who have received an allogeneic hematopoeitic cell transpant, we generally describe haplotect contamination estimates of 2%--5% as evidence low-level chimerism and estimates \>5% as evidence of significant chimerism.

MyeloSeq is also ordered on peripheral blood specimens for patients with chronic unexplained cytopenias, particularly in the outpatient setting. Detecting mutations in these patients demonstrates they have clonal hematopoiesis, but it is generally not sufficient to establish a specific diagnosis, which usually requires a bone marrow biopsy.

Examples:

-   *Mutations in BCOR and DNMT3A are identified, consistent with the diagnosis of a myeloid neoplasm. The differential diagnosis includes clonal cytopenia of undetermined significance (CCUS) and myelodysplastic syndrome (MDS). A bone marrow biopsy is required for further evaluation.*

-   *No mutations are identified. Overall quality control metrics are within normal limits. The absence of a mutation detectable with this assay does not rule out a diagnosis of myelodysplastic syndrome.*

### Uploading Cases to CoPath

Once the final set of reportable variants have been selected and the `Variant Interpretation`, `Clinical History`, and `Specimen Site` have all been entered, clicking `Generate and download RTF` saves a formatted version of the report as an rtf document to your default downloads location. Note that some default applications for view `*.rtf` files disrupts the structure of the tables. Please open the `*.rtf` files with Microsoft Word to display them correctly. The report includes blue headers corresponding to the structured sections of the CoPath report: `Interpretation`, `Diagnosis`, `Results-Commments`. To upload the data to CoPath:

1.  Open the corresponding case in CoPath (select `Procedure Entry/Addendum Entry/Edit`).
2.  Enter the name of the attending assigned to the case as well as your own under `Procedure Pathologist(s)`.
3.  Click `Edit Text`, which opens a CoPath instance of Microsoft Word. **Always confirm that the case accession at the top of the CoPath window exactly matches the case accession in the `*.rtf` document.**
4.  Copy and paste the data (excluding the headers) into each section of the CoPath Word document. Note that by default, text is pasted into CoPath without formatting. Use **Option**-**Shift**-**Command**-**V** on a PC or **Option**-**Control**-**Command**-**V** on a Mac to preserve the text formatting when pasting the data in CoPath. Also, note that the table widths are not cleanly formatted by default. Please extend the right border of the table so that it occupies the full width of the page and arrange the columns so that the content is clean and easy to read.
5.  Once you've finished editing the report, click the pink box in the upper left corner to exit the CoPath Word application.
6.  Click `Save/Next Specimen`
7.  If the case is ready to be reviewed by the attending, place it in his/her queue by selecting `Complete/Send to Pathologist`; otherwise, if you are still working on the case, select `No Action`.
8.  Under "Do you want to save changes to specimen W##-####?" click `Yes` to save your changes; **your work is not saved until you click** `Yes`.

### References

#### Background

-   Duncavage EJ *et al.* Genomic profiling for clinical decision making in myeloid neoplasms and acute leukemia. *Blood.* 2022 Nov 24;140(21):2228-2247. [PMID: 36130297](https://pubmed.ncbi.nlm.nih.gov/36130297/).

#### Disease Classification

-   Arber DA *et al.* International Consensus Classification of Myeloid Neoplasms and Acute Leukemias: integrating morphologic, clinical, and genomic data. *Blood.* 2022 Sep 15;140(11):1200-1228. [PMID: 35767897](https://pubmed.ncbi.nlm.nih.gov/35767897/).

-   Khoury JD *et al.* The 5th edition of the World Health Organization Classification of Haematolymphoid Tumours: Myeloid and Histiocytic/Dendritic Neoplasms. *Leukemia.* 2022 Jul;36(7):1703-1719. [PMID: 35732831](https://pubmed.ncbi.nlm.nih.gov/35732831/).

#### NCCN Guidelines

**Please refer to the most recent guidelines published on the [NCCN](https://www.nccn.org/guidelines/) website, which may include updates to the references highlighted below.**

-   Gerds AT *et al*. Myeloproliferative Neoplasms, Version 3.2022, NCCN Clinical Practice Guidelines in Oncology. *J Natl Compr Canc Netw.* 2022 Sep;20(9):1033-1062. [PMID: 36075392](https://pubmed.ncbi.nlm.nih.gov/36075392/).

-   Greenberg PL *et al.* NCCN Guidelines® Insights: Myelodysplastic Syndromes, Version 3.2022. *J Natl Compr Canc Netw.* 2022 Feb;20(2):106-117. [PMID: 35130502](https://pubmed.ncbi.nlm.nih.gov/35130502/).

-   Pollyea DA *et al.* Acute Myeloid Leukemia, Version 3.2023, NCCN Clinical Practice Guidelines in Oncology. *J Natl Compr Canc Netw.* 2023 May;21(5):503-513. [PMID: 37156478](https://pubmed.ncbi.nlm.nih.gov/37156478/).

#### Special Cases

-   Beck DB *et al.* Somatic Mutations in *UBA1* and Severe Adult-Onset Autoinflammatory Disease. *N Engl J Med.* 2020 Dec 31;383(27):2628-2638. [PMID: 33108101](https://pubmed.ncbi.nlm.nih.gov/33108101).

-   Jongen-Lavrencic M *et al.* Molecular Minimal Residual Disease in Acute Myeloid Leukemia. *N Engl J Med.* 2018 Mar 29;378(13):1189-1199. [PMID: 29601269](https://pubmed.ncbi.nlm.nih.gov/29601269).

-   Li P *et al.* The genetic landscape of germline *DDX41* variants predisposing to myeloid neoplasms. *Blood.* 2022 Aug 18;140(7):716-755. [PMID: 35671390](https://pubmed.ncbi.nlm.nih.gov/35671390/).

-   Taube F *et al.* *CEBPA* mutations in 4708 patients with acute myeloid leukemia: differential impact of bZIP and TAD mutations on outcome. *Blood.* 2022 Jan 6;139(1):87-103. [PMID: 34320176](https://pubmed.ncbi.nlm.nih.gov/34320176/).
