<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Drew Hughes, MD, PhD - GatewaySeq: Introduction for Trainees</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Drew Hughes, MD, PhD</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html" rel="" target="">
 <span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./resources.html" rel="" target="">
 <span class="menu-text">Resources</span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools ms-auto">
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#overview" id="toc-overview" class="nav-link active" data-scroll-target="#overview">Overview</a></li>
  <li><a href="#quick-tips" id="toc-quick-tips" class="nav-link" data-scroll-target="#quick-tips">Quick Tips</a></li>
  <li><a href="#workflow" id="toc-workflow" class="nav-link" data-scroll-target="#workflow">Workflow</a></li>
  <li><a href="#data" id="toc-data" class="nav-link" data-scroll-target="#data">Data</a></li>
  <li><a href="#reporting-tool" id="toc-reporting-tool" class="nav-link" data-scroll-target="#reporting-tool">Reporting Tool</a></li>
  <li><a href="#case" id="toc-case" class="nav-link" data-scroll-target="#case">Case</a></li>
  <li><a href="#qc" id="toc-qc" class="nav-link" data-scroll-target="#qc">QC</a></li>
  <li><a href="#variants" id="toc-variants" class="nav-link" data-scroll-target="#variants">Variants</a></li>
  <li><a href="#cnv" id="toc-cnv" class="nav-link" data-scroll-target="#cnv">CNV</a></li>
  <li><a href="#fusions" id="toc-fusions" class="nav-link" data-scroll-target="#fusions">Fusions</a></li>
  <li><a href="#tmbmsi" id="toc-tmbmsi" class="nav-link" data-scroll-target="#tmbmsi">TMB/MSI</a></li>
  <li><a href="#interpretation" id="toc-interpretation" class="nav-link" data-scroll-target="#interpretation">Interpretation</a></li>
  <li><a href="#pdf" id="toc-pdf" class="nav-link" data-scroll-target="#pdf">PDF</a></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references">References</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">GatewaySeq: Introduction for Trainees</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<style>title {text-align: left}</style>
<style>body {text-align: justify}</style>
<section id="overview" class="level3">
<h3 class="anchored" data-anchor-id="overview">Overview</h3>
<p><a href="https://pathologyservices.wustl.edu/items/gatewayseq-ngs-panel-with-interpretation/">GatewaySeq</a> is a tumor-only, DNA-only, targeted next-generation sequencing assay for the identification of clinically significant genetic alterations in solid tumors for diagnostic, prognostic, and therapeutic purposes. GatewaySeq is designed to evaluate key molecular biomarkers at diagnosis, relapse, or metastasis. The assay identifies small variants (single nucleotide variants, insertions, and deletions), copy number alterations (CNAs), and structural variants. GatewaySeq also evaluates microsatellite instability (MSI) and tumor mutational burden (TMB). The limit of detection (LOD) for GatewaySeq is 5% variant allele fraction (VAF), which depends on both the abundance of individual variants as well as tumor cellularity (minimum 5%). Formalin- or alcohol-fixed, paraffin-embedded surgical or cytology specimens (cell blocks only) are acceptable for testing. GatewaySeq was designed for the following tumor types: bone and soft tissue, brain, breast, gastrointestinal tract, genitourinary tract, gynecologic tract, hepatobiliary tract, lymphoma, lung, skin, thyroid.</p>
<p>GatewaySeq can be ordered at the discretion of the provider (via Epic) or by the pathologist reviewing the surgical or cytology case (via Copath). All new diagnoses of non-small cell lung cancer should have GatewaySeq ordered automatically by the pathologist.</p>
</section>
<section id="quick-tips" class="level3">
<h3 class="anchored" data-anchor-id="quick-tips">Quick Tips</h3>
<ul>
<li><strong>Read this entire document and refer to it while templating cases</strong></li>
<li><strong>Review examples of previous cases, especially interpretations (this can be done using the GatewaySeq reporting tool)</strong></li>
<li><strong>Always incorporate the clinical and pathological context into the interpretation of individual variants as well as the overall case</strong></li>
<li><strong>Never click <code>Save JSON + ProcessPdf</code> within the <code>PDF</code> tab; this is for the sign out pathologist only</strong></li>
<li><strong>You must click <code>Save JSON</code> within the <code>PDF</code> tab to save your work; none of your changes are saved until you click this button; if you return to the <code>Orders</code> tab or close your browser window before clicking <code>Save JSON</code>, your changes will be lost</strong></li>
</ul>
</section>
<section id="workflow" class="level3">
<h3 class="anchored" data-anchor-id="workflow">Workflow</h3>
<p>GatewaySeq is currently run once per week. Sequencing libraries are loaded on a NovaSeq X Plus on Friday, and data are typically available for analysis by Monday morning. Case assignments are distributed via email by the fellow on service on Friday. <strong>Trainees review the sequencing data and template clinical reports for sign out by 1400 on Monday afternoon.</strong> Briefly, trainees are expected to:</p>
<ul>
<li>Review the clinical history, especially the corresponding and historical pathology case(s), and incorporate those findings into the case interpretation</li>
<li>Review any previous molecular testing performed on the same or related sample(s) (cytogenetics, external next-generation sequencing, etc.) and compare to GatewaySeq results</li>
<li>Review quality control data and <strong>notify the attending</strong> of any cases that don’t meet target thresholds</li>
<li>Evaluate and classify individual small variants, CNAs, and SVs</li>
<li>Compose a brief interpretation for each clinically significant variant</li>
<li>Verify classification of MSI and TMB</li>
<li>Compose a brief interpretation summarizing key results for the overall case</li>
<li>Save the updated data within the GatewaySeq reporting tool</li>
<li>Review the case with sign out faculty and make revisions as needed</li>
</ul>
</section>
<section id="data" class="level3">
<h3 class="anchored" data-anchor-id="data">Data</h3>
<p>The complete set of files for each case are stored on the Research Infrastructure Services (RIS) cluster (ris.wustl.edu). The subset of files needed to interpret most cases are copied to the Department of Pathology server (pathfiles.wustl.edu). You do not need to routinely access these files directly to template cases; most of this information is available within the reporting tool. However, if you need to investigate individual variants or otherwise need access to data not included in the reporting tool, these files are available.</p>
<p>To connect to the department server on a Windows computer, use the File Explorer application:</p>
<ul>
<li>Right click on <code>This PC</code> and select <code>Map network drive...</code></li>
<li>Type <code>\\pathfiles.wustl.edu\AMP\Genomics\GatewaySeq</code> into the <code>Folder</code> box and click <code>Finish</code></li>
</ul>
<p>To access data on a Mac, use the Finder application:</p>
<ul>
<li>Select <code>Go</code> → <code>Connect to Server</code> from the menu bar (or press <code>⌘K</code>)</li>
<li>Type <code>smb://pathfiles.wustl.edu/AMP/Genomics/GatewaySeq</code> into the Server Address box and click Connect</li>
</ul>
<p>Cases are organized on the department server by batch. Each batch directory contains a separate folder for each case as well as a TSV file summarizing the quality control metrics for all cases (<code>QC_metrics.tsv</code>). Each case folder contains the following files:</p>
<ul>
<li><code>*annotated.vcf.gz</code>: variant call format file of all potentially reportable small variants detected by the analysis pipeline</li>
<li><code>*annotated.vcf.gz.tbi</code>: small variant VCF index</li>
<li><code>*sv_annotated.vcf.gz</code>: variant call format file of all potentially reportable structural variants detected by the analysis pipeline</li>
<li><code>*sv_annotated.vcf.gz.tbi</code>: structural variant VCF index</li>
<li><code>dragen/*cram</code>: alignment file</li>
<li><code>dragen/*cram.crai</code>: alignment index</li>
<li><code>*report.json</code>: case report text file`</li>
<li><code>*report.txt</code>: case report json file</li>
</ul>
</section>
<section id="reporting-tool" class="level3">
<h3 class="anchored" data-anchor-id="reporting-tool">Reporting Tool</h3>
<p>The <a href="https://mopath.wustl.edu/gatewayseq/">GatewayReporter</a> tool is a web-based JavaScript application used to prepare GatewaySeq clinical reports. This tool reads data from the .json files associated with each case. The landing page (<code>Orders</code> tab) presents a dashboard of pending and completed cases (completed cases can be viewed by de-selecting <code>Hide Final orders</code>). Clicking on an individual <code>orderid</code> in this table allows you to enter and review and/or edit a specific case. You can then work through all of the components of the case by moving left to right across the tabs in the reporting tool (<code>Case</code>, <code>QC</code>, <code>Variants</code>, <code>CNV</code>, <code>Fusions</code>, <code>TMB/MSI</code>, <code>Interpretation</code>, <code>PDF</code>, and <code>JSON</code>).</p>
</section>
<section id="case" class="level3">
<h3 class="anchored" data-anchor-id="case">Case</h3>
<p><strong>Patient Information</strong></p>
<ul>
<li>The patient <code>Name</code>, <code>Date Of Birth</code>, <code>Sex</code>, <code>MRN</code>, and CoPath <code>Accession Number</code> are populated directly from Copath; verify these for accuracy</li>
<li><code>Ordering Physician</code>: the attending physician on the associated Epic order or the ordering pathologist for orders placed by pathology (Epic requisitions and/or histology orders are scanned into the Image Gallery in CoPath)</li>
</ul>
<p><strong>Specimen Information</strong></p>
<ul>
<li><code>Tumor Origin</code>: select from dropdown</li>
<li><code>Specimen Site</code>, <code>Corresponding Specimen Number</code>, and <code>Specimen Collected Date</code>: enter based on corresponding surgical or cytology case</li>
<li><code>Block number</code>: enter from histology order</li>
<li><code>Tumor cellularity</code>: enter from histology order; if not listed, ask the Molecular Coordinators to pull the case to assess tumor cellularity</li>
</ul>
<p><strong>Diagnostic Information</strong></p>
<ul>
<li><code>Stated Diagnosis</code>: enter from corresponding surgical or cytology case</li>
<li><code>Diagnostic Timepoint</code>, <code>Histologic Grade</code>, <code>T stage</code>, <code>N stage</code>, <code>M stage</code>, and <code>AJCC/FIGO Stage Group</code>: select from dropdown based on corresponding surgical or cytology case; if information is not known, select NA</li>
</ul>
</section>
<section id="qc" class="level3">
<h3 class="anchored" data-anchor-id="qc">QC</h3>
<p>Cases with any of QC metrics outside of the reference range may require additional sequencing, preparation of new sequencing libraries, or submission of alternative specimens. <strong>Any cases that do not meet QC metrics should be brought to the attention of the sign out pathologist as soon as possible.</strong></p>
<ul>
<li><code>Mapping/Aligning Summary</code>: sequencing yield, read quality, alignment rate, and insert size</li>
<li><code>Coverage Summary</code>: on-target rate, on-target coverage</li>
<li><code>UMI Summary</code>: percentage of duplicate reads (reflects complexity of input material/number of distinct input molecules)</li>
<li><code>Failed Exon QC</code>: small variant target regions that did not have adequate mean coverage or an adequate proportion of target bases covered at &gt;100x or &gt;250x</li>
<li><code>Gene Coverage QC</code>: mean coverage, proportion of target bases covered at &gt;100x, and proportion of target bases covered at &gt;250x for every gene on the panel</li>
<li><code>Fusion Coverage QC</code>: fusion target regions that did not have adequate mean coverage or an adequate proportion of target bases covered at &gt;100x or &gt;250x</li>
<li><code>MSI/Haplotect coverage QC</code>: MSI and haplotect target regions that did not have adequate mean coverage or an adequate proportion of target bases covered at &gt;100x or &gt;250x</li>
<li><code>Haplotect summary</code>: haplotect is an algorithm that analyzes known polymorphic sites in the genome to quantitatively estimate sample contamination (i.e., evidence of the presence of DNA from more than one individual in the sequencing data); if the contamination estimate is greater than 0.04 with &gt;1 informative site, there is evidence of potential contamination, which must be accounted for during the interpretation, and/or a new specimen or new library may be needed</li>
</ul>
</section>
<section id="variants" class="level3">
<h3 class="anchored" data-anchor-id="variants">Variants</h3>
<p>Variant classification and interpretation are based on the AMP/ASCO/CAP guidelines for reporting sequence variants in cancer (<a href="https://pubmed.ncbi.nlm.nih.gov/27993330/">PMID: 27993330</a>). Tier 1 and 2 variants are classified as Significant and Tier 3 variants are classified as Variants of Uncertain Significance (VUS). Tier 4 variants are not reported.</p>
<p>Variants that pass all internal QC filters are listed in the <code>PASS</code> table. Variants in genes with potential clinical significance for each tumor type (based on the data entered on the <code>Case</code> tab) are classified as Significant by default. Review these classifications to ensure that variants classified as Significant meet criteria for Tier 1 or 2 variants and that variants classified as VUS do not (use the checkboxes to update the classifications as needed). Note that our internal reporting policy is to classify Oncogenic/Likely Oncogenic variants as Tier 2. Additional information for each variant, including links to external databases, can be viewed by clicking on the <strong><code>⋁</code></strong> symbol. The majority of Tier 4 variants are automatically filtered from the report based on their frequency in the general population (filtering frequency &gt;0.1%); however, the annotation and comparison of multinucleotide variants to population databases in particular is error-prone and should be double-checked to ensure that Tier 4 variants aren’t mistakenly reported as VUS.</p>
<p>Variants that are below the validated LOD (5% VAF) or that fail other quality filters are listed in the <code>Filtered</code> table. Variants included here should rarely be reported. The primary exceptions are 1) variants that represent a nucleotide change that is identical to a variant that was previously reported in the same patient by orthogonal testing and whose only <code>Filter</code> flag is <code>LowReads</code>, and 2) variants with strong clinical significance that are just below the LOD and whose only <code>Filter</code> flag is <code>LowReads</code> (e.g., an <em>EGFR</em> exon 21 variant at a VAF of 4.8% in a lung cancer specimen with low tumor cellularity).</p>
</section>
<section id="cnv" class="level3">
<h3 class="anchored" data-anchor-id="cnv">CNV</h3>
<p>Copy number variants (CNVs) with copy ratios ≥2.0 (gain) or ≤0.5 (loss) are reported. Occasionally, CNVs will be listed in the <code>Filtered</code> table with a high/low copy ratio but whose only filter is <code>cnvLength</code>; these should usually be promoted to significant and included in the report.</p>
</section>
<section id="fusions" class="level3">
<h3 class="anchored" data-anchor-id="fusions">Fusions</h3>
<p>Fusions in the <code>PASS</code> table should usually be reported. Fusions listed in the <code>Filtered</code> table should be manually reviewed by <a href="https://genome.ucsc.edu/cgi-bin/hgBlat">BLATing</a> the contig to ensure it represents a clean junction between two segments of high-quality sequence with high-quality alignments. <code>Filtered</code> fusion calls with only single read support (<code>NoPairedReads</code> flag and <code>PR_READS=0</code> in the Info field) are usually artifacts.</p>
</section>
<section id="tmbmsi" class="level3">
<h3 class="anchored" data-anchor-id="tmbmsi">TMB/MSI</h3>
<p>Non-synonymous TMB (mutations/Mb) is reported with a categorization: low (0–10 mutations/Mb), intermediate (11–20 mutations/Mb), and high (&gt;20 mutations/Mb).</p>
<p>Cases with <code>PercentageUnstableSites</code> &lt;20 are classified as microsatellite stable (MSS) and cases with <code>PercentageUnstableSites</code> ≥20 are classified as microsatellite instability high (MSI-H). If <code>ResultIsValid</code> is <code>false</code>, there is a technical problem with the MSI anlaysis and the sign out pathologist should be notified.</p>
</section>
<section id="interpretation" class="level3">
<h3 class="anchored" data-anchor-id="interpretation">Interpretation</h3>
<p><strong>Clinically Significant Genes with No Detected Variants</strong></p>
<p>This section is automatically generated based on the tumor type and the variants selected as significant.</p>
<p><strong>Interpretation for Clinically Significant Variants</strong></p>
<p>Each variant that was selected as Significant in the <code>Variants</code> tab will appear here. At least one of the four significance categories should be selected (Diagnostic, Prognostic, Therapeutic, and/or Oncogenic/Likely Oncogenic).</p>
<p>The reporting tool includes disease-specific interpretations from Memorial Sloan Kettering Cancer Center’s precision oncology knowledge base, <a href="https://www.oncokb.org/">OncoKB</a>, for each variant. Use the dropdowns to expand the tree under the <code>OncoKB</code> column and select the appropriate disease and click on the green text to pull comments into the <code>Interpretation</code> field. The comments entered for <code>geneSummary</code>, <code>variantSummary</code>, and <code>tumorTypeSummary</code> should generally be included in the interpretation and revised as needed. Please italicize all gene names by placing them inside asterisks (e.g., *EGFR*). Regarding variants with potential therapeutic significance, our current policy is to include FDA-approved and/or NCCN-compendium listed treatments (i.e., not investigational agents) in the written interpretation.</p>
<p>The OncoKB annotations typically provide high quality information for recurrent variants/variant types in key genes, but additional investigation is needed to interpret less well-characterized variants. The following resources may be helpful in those cases: <a href="https://pubmed.ncbi.nlm.nih.gov">Pubmed</a>, <a href="https://franklin.genoox.com/clinical-db/home">Franklin</a>, <a href="https://cancer.sanger.ac.uk/cosmic">COSMIC</a>, and <a href="https://civicdb.org">CIViC</a>.</p>
<p><strong>Interpretation for Included CNAs</strong> and <strong>Interpretation for Included Fusions</strong></p>
<p>Similar to the interpretations for individual small variants, interpretations for CNAs or Fusions included in the report should be entered here. These fields are not prepopulated with OncoKB annotations; however, the <a href="https://www.oncokb.org/">OncoKB</a> web portal provides information for many of the recurrent CNAs and Fusions that are detected by GatewaySeq and may be a helpful resource when collecting data about these variants.</p>
<p><strong>Overall interpretation</strong></p>
<p>Provide a brief (2–3 sentence) summary of the entire case. The goal of the overall interpretation is to summarize the clinical significance of variants observed with an emphasis on implications for diagnosis, prognosis, and treatment. Relevant negatives can also be included here, especially when the provider has indicated they are interested in the presence or absence of specific variants.</p>
<p>For cases with no variants that provide diagnostic, prognostic, or therapeutic information, we generally provide the following comment:</p>
<p><em>No variants associated with FDA-approved or NCCN-compendium listed treatments are identified. Overall quality control metrics are within normal limits.</em></p>
</section>
<section id="pdf" class="level3">
<h3 class="anchored" data-anchor-id="pdf">PDF</h3>
<p>After the QC review is complete and the case data and variant interpretations have been entered, selecting the <code>PDF</code> tab renders a preview of the final PDF report. Proofread this draft carefully; it is identical to the report that will appear in CoPath and Epic. Return to relevant tabs to make any necessary edits, and then return to the <code>PDF</code> tab to review any changes and save your work.</p>
<p><strong>Do not click <code>Save JSON + ProcessPdf</code>. This should only be clicked by the sign out pathologist.</strong> Clicking on this button finalizes the case and sends it to CoPath, preventing further changes without issuing an amended report.</p>
<p><strong>You must click <code>Save JSON</code> to save your work (across all tabs).</strong> After clicking this, the next time the case is loaded from the <code>Orders</code> tab, the case will load with the updated information.</p>
</section>
<section id="references" class="level3">
<h3 class="anchored" data-anchor-id="references">References</h3>
<section id="general" class="level4">
<h4 class="anchored" data-anchor-id="general">General</h4>
<ul>
<li><a href="https://www.nccn.org">National Comprehensive Cancer Network (NCCN) Guidelines</a></li>
<li><a href="https://www.oncokb.org">OncoKB</a></li>
</ul>
</section>
<section id="endometrial-cancer" class="level4">
<h4 class="anchored" data-anchor-id="endometrial-cancer">Endometrial Cancer</h4>
</section>
<section id="lymphoma" class="level4">
<h4 class="anchored" data-anchor-id="lymphoma">Lymphoma</h4>
<ul>
<li><p>de Leval L <em>et al.</em> Genomic profiling for clinical decision making in lymphoid neoplasms. <em>Blood.</em> 2022 Nov 24;140(21):2193-2227. <a href="https://pubmed.ncbi.nlm.nih.gov/36001803/">PMID: 36001803</a>.</p></li>
<li><p>Nann D <em>et al.</em> Follicular lymphoma t(14;18)-negative is genetically a heterogeneous disease. <em>Blood Adv.</em> 2020 Nov 24;4(22):5652-5665. <a href="https://pubmed.ncbi.nlm.nih.gov/33211828/">PMID: 33211828</a>.</p></li>
<li><p>Salaverria I <em>et al.</em> The clinical and molecular taxonomy of t(14;18)-negative follicular lymphomas. <em>Blood Adv.</em> 2023 Sep 26;7(18):5258-5271. <a href="https://pubmed.ncbi.nlm.nih.gov/37561599/">PMID: 37561599</a>.</p></li>
<li><p>Sánchez-Beato M <em>et al.</em> A genetic profiling guideline to support diagnosis and clinical management of lymphomas. <em>Clin Transl Oncol.</em> 2023 Sep 6. <a href="https://pubmed.ncbi.nlm.nih.gov/37672206/">PMID: 37672206</a>.</p></li>
</ul>
<p><br>
</p>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  let localAlternateSentinel = 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>